client/hosting/server-settings/components/cache-card/index.tsx

```diff
@@ -88,102 +88,123 @@ export const CacheCard = ( {
 			headingId="cache"
 			title={ translate( 'Performance optimization' ) }
 		>
-			<div>
```

Comment:
I just removed `div`, that's why a lot of changes

Reply:
Could we implement this change in https://github.com/Automattic/wp-calypso/pull/94664, too? That would make the diff easier to read